src/Symfony/Bundle/TwigBundle/composer.json

```diff
@@ -20,7 +20,7 @@
         "composer-runtime-api": ">=2.1",
         "symfony/config": "^6.4|^7.0",
         "symfony/dependency-injection": "^6.4|^7.0",
-        "symfony/twig-bridge": "^6.4|^7.0",
+        "symfony/twig-bridge": "^7.2",
```

Comment:
@nicolas-grekas I'm not entirely sure about this one, is that correct ?
We need to have an up-to-date Twig bridge available to get the new Caches.